indico/modules/categories/operations.py

```diff
@@ -51,12 +51,7 @@ def move_category(category, target_category):
 
 
 @make_interceptable
-def update_category(category, data, skip=()):
-    assert set(data) <= {
-        'title', 'description', 'timezone', 'suggestions_disabled', 'is_flat_view_enabled',
-        'event_message_mode', 'event_message', 'notify_managers', 'event_creation_notification_emails',
-        *skip
-    }
+def update_category(category, data, skip=(), _extra_log_fields={}):
```

Comment:
Do not use mutable default values - use `None` here and `whatever or {}` when you need a dict.